src/test/java/unit/SshKeyPair.java

```diff
@@ -0,0 +1,63 @@
+package unit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import bio.terra.cli.serialization.userfacing.UFSshKeyPair;
+import harness.TestCommand;
+import harness.TestUser;
+import harness.baseclasses.SingleWorkspaceUnit;
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("unit")
+public class SshKeyPair extends SingleWorkspaceUnit {
+
+  TestUser testUser;
+
+  @BeforeAll
+  @Override
+  protected void setupOnce() throws Exception {
+    super.setupOnce();
+    testUser = TestUser.chooseTestUser();
+    testUser.login();
+  }
+
+  @Test
+  void getSshKey() throws IOException {
+    testUser.login();
```

Comment:
`testUser` is already logged in. Similarly below.

Reply:
realized that i actually need to keep this in the test. loggin in in the setup doesn't work